frontend/www/js/omegaup/problem/edit.ts

```diff
@@ -1,46 +1,239 @@
-import { OmegaUp } from '../omegaup';
+import { OmegaUp, omegaup } from '../omegaup';
 import { types } from '../api_types';
 import T from '../lang';
 import Vue from 'vue';
-import problem_Edit from '../components/problem/Form.vue';
+import problem_Edit from '../components/problem/Edit.vue';
 import * as ui from '../ui';
 import * as api from '../api';
+import * as Markdown from '@/third_party/js/pagedown/Markdown.Converter.js';
 
 OmegaUp.on('ready', () => {
-  const payload = types.payloadParsers.ProblemFormPayload(
-    'problem-edit-payload',
-  );
+  const payload = types.payloadParsers.ProblemEditPayload();
   if (payload.statusError) {
     ui.error(payload.statusError);
   }
+  if (payload.statusSuccess) {
+    ui.success(T.problemEditUpdatedSuccessfully);
+  }
+  const lang = payload.statement.language;
+  const statements: types.Statements = {};
+  const solutions: types.Statements = {};
+  statements[lang] = payload.statement.markdown;
+  solutions[lang] = payload.solution.markdown;
```

Comment:
hay manera de hacer esto de manera corta!

```suggestion
  const statements: types.Statements = {
    [payload.statement.language]: payload.statement.markdown,
  };
  const solutions: types.Statements = {
    [payload.statement.language]: payload.solution.markdown,
  };
```

Reply:
Listo.